refextract/references/kbs.py

```diff
@@ -42,6 +43,23 @@
 from ..documents.text import re_group_captured_multiple_space
 
 
+@contextlib.contextmanager
+def file_resolving(fpath, reader=None, **kwargs):
+    if isinstance(fpath, six.string_types):
+        print('Load kb path %s' % fpath)
```

Comment:
@jacquerie I agree and there cases here that can be removed. So, I will leave only a logger message inside the contextmanager and I will delete all the other messages. And if more log messages are needed, I can create another PR to introduce them.